Apps/frontend/src/app.scss

```diff
@@ -208,6 +218,10 @@ nav {
     }
   }
 }
+.margin-small {
```

Comment:
Do you think we'll need `margin-medium` or sth. like that in the future? If you do, can you rewrite the class into e.g. `.margin {
  &--small {
    margin-right: 5px;
    margin-left: 5px;
  }
}
`

Reply:
Don't think so but I adapted it